src/type-generator.ts

```diff
@@ -521,6 +521,8 @@ export class TypeGenerator {
 
       code.openBlock(`export enum ${typeName}`);
 
+      const processedValues: string[] = [];
```

Comment:
nit: can we make this a set for quick lookup?

Reply:
Good catch. On it.